src/styles/modules/_m-checkbox.sass

```diff
@@ -38,7 +38,7 @@
   +e(label-container)
     display: flex
     width: 100%
-    margin-left: .5rem
+    margin-left: 14px
```

Comment:
This is looking good, but, as with the RA codebase,  we need to use the rem-fallback mixin.  So it should look like this:
`@include rem-fallback('margin-left', 14px);`

Reply:
Fixed

Reply:
@sgashler this fix isn't quite right.  The extra "margin-left:" isn't needed.  It should be replaced with just `@include rem-fallback('margin-left', 14px)`

Reply:
Fixed ... for reals this time.